packages/driver/src/dom/elements/find.ts

```diff
@@ -125,10 +125,16 @@ export const getFirstDeepestElement = ($el: JQuery, index = 0) => {
   const $current = $el.slice(index, index + 1)
   const $next = $el.slice(index + 1, index + 2)
 
-  if (!$next) {
+  if (!$next || $current.length === 0) {
     return $current
   }
 
+  // https://github.com/cypress-io/cypress/issues/19377
```

Comment:
Lets link to the original issue this change was meant to solve: 
```suggestion
  // https://github.com/cypress-io/cypress/issues/14861
```